results.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure that this file was ever meant to be checked in to begin with, looks like test results.